localisation/es-es/README.md

```diff
@@ -103,4 +103,7 @@ Revisa el ejemplo completo de cada aplicación. También están programadas con
 - [GitHub Copilot: MCP](https://code.visualstudio.com/blogs/2025/05/12/agent-mode-meets-mcp)
 - [GitHub Copilot: Instrucciones Personalizadas](https://code.visualstudio.com/docs/copilot/copilot-customization)
 - [GitHub Copilot: Cambiar Modelos de IA](https://docs.github.com/en/copilot/using-github-copilot/ai-models/changing-the-ai-model-for-copilot-chat?tool=vscode)
-- [Servidores MCP Curados](https://github.com/modelcontextprotocol/servers)
\ No newline at end of file
+- [Servidores MCP Curados](https://github.com/modelcontextprotocol/servers)
+---
+
+Este documento ha sido localizado por [GitHub Copilot](https://docs.github.com/copilot/about-github-copilot/what-is-github-copilot). Por lo tanto, puede contener errores. Si encuentras alguna traducción que sea inapropiada o errónea, por favor crea un [issue](../../issues).
```

Comment:
Add `**Disclaimer**: ` in th beginning of this disclaimer sentence in the given language.